### PR TITLE
Add Final Fantasy Explorers

### PR DESCRIPTION
**Pull request by Genjar**
Adds Final Fantasy Explorers.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/final-fantasy-explorers/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "Final Fantasy Explorers"
+description = "Final Fantasy x Monster Hunter mashup, with tameable monster allies."
```
Comment (robzombie91) [inline games/final-fantasy-explorers/game.dat]: Final Fantasy Explorers is an action role-playing video game developed by Racjin and published by Square Enix for the Nintendo 3DS. It features character job-oriented combat against classic Final Fantasy monsters and summons.

File: games/final-fantasy-explorers/game.dat [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+needs_system_files = false
+needs_shared_font = false
+
+[[releases]]
```
Comment (robzombie91) [inline games/final-fantasy-explorers/game.dat]: NA - 000400000016ED00, 2016-01-26
JP - 000400000010E200, 2014-12-18

**Comment by robzombie91** [comment]
Also please use the American Boxart if at ever possible : [Here](https://art.gametdb.com/3ds/coverHQ/US/BCEE.jpg?1456936891)

File: games/final-fantasy-explorers/game.dat [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+title = "000400000016E700"
+region = "eur"
+release_date = "2016-01-26"
```
Comment (Leo626) [inline games/final-fantasy-explorers/game.dat]: Wrong date

> EU: January 29, 2016[

Comment (robzombie91) [reply games/final-fantasy-explorers/game.dat]: whoops my bad

**Comment by Leo626** [comment]
Superseded by #287.

**Pull request closed without merging**